NOTES.md

```diff
@@ -1,3 +1,24 @@
+# Release 1.16.1 (Stand By Me)
+
+Released on 2015-02-12
+
+Bug fix update.
+
+* Now write to the log every time a file is deleted (#1780)
+* Fixed `rethinkdb dump` and other backup scripts to correctly detect the server version (#3706)
+* Changed the output of `rethinkdb dump` to clarify that indexes are being saved (#3708)
+* Fixed memory consumption with newer versions of boost (#3712)
```

Comment:
We should probably mention that this affected our OS X dmg.
How about
"Fixed unbounded memory consumption when using the official OS X package or when building with boost version 1.56.0 or higher"